bwapi/Backend/libprotobuf-lite/libprotobuf-lite.vcxproj

```diff
@@ -103,53 +103,53 @@
+    <ClCompile Include="src\google\protobuf\generated_message_util.cc" />
+    <ClCompile Include="src\google\protobuf\implicit_weak_message.cc" />
+    <ClCompile Include="src\google\protobuf\io\coded_stream.cc" />
+    <ClCompile Include="src\google\protobuf\io\zero_copy_stream.cc" />
+    <ClCompile Include="src\google\protobuf\io\zero_copy_stream_impl_lite.cc" />
+    <ClCompile Include="src\google\protobuf\message_lite.cc" />
+    <ClCompile Include="src\google\protobuf\repeated_field.cc" />
+    <ClCompile Include="src\google\protobuf\stubs\bytestream.cc" />
+    <ClCompile Include="src\google\protobuf\stubs\common.cc" />
+    <ClCompile Include="src\google\protobuf\stubs\int128.cc" />
+    <ClCompile Include="src\google\protobuf\stubs\io_win32.cc" />
+    <ClCompile Include="src\google\protobuf\stubs\status.cc" />
+    <ClCompile Include="src\google\protobuf\stubs\statusor.cc" />
+    <ClCompile Include="src\google\protobuf\stubs\stringpiece.cc" />
+    <ClCompile Include="src\google\protobuf\stubs\stringprintf.cc" />
+    <ClCompile Include="src\google\protobuf\stubs\structurally_valid.cc" />
+    <ClCompile Include="src\google\protobuf\stubs\strutil.cc" />
+    <ClCompile Include="src\google\protobuf\stubs\time.cc" />
+    <ClCompile Include="src\google\protobuf\wire_format_lite.cc" />
+    <ClInclude Include="src\google\protobuf\arena.h" />
+    <ClInclude Include="src\google\protobuf\arenastring.h" />
+    <ClInclude Include="src\google\protobuf\extension_set.h" />
+    <ClInclude Include="src\google\protobuf\generated_message_util.h" />
+    <ClInclude Include="src\google\protobuf\implicit_weak_message.h" />
+    <ClInclude Include="src\google\protobuf\io\coded_stream.h" />
+    <ClInclude Include="src\google\protobuf\io\zero_copy_stream.h" />
+    <ClInclude Include="src\google\protobuf\io\zero_copy_stream_impl_lite.h" />
+    <ClInclude Include="src\google\protobuf\message_lite.h" />
+    <ClInclude Include="src\google\protobuf\repeated_field.h" />
+    <ClInclude Include="src\google\protobuf\stubs\bytestream.h" />
+    <ClInclude Include="src\google\protobuf\stubs\common.h" />
+    <ClInclude Include="src\google\protobuf\stubs\int128.h" />
+    <ClInclude Include="src\google\protobuf\stubs\once.h" />
+    <ClInclude Include="src\google\protobuf\stubs\status.h" />
+    <ClInclude Include="src\google\protobuf\stubs\statusor.h" />
+    <ClInclude Include="src\google\protobuf\stubs\stringpiece.h" />
+    <ClInclude Include="src\google\protobuf\stubs\stringprintf.h" />
+    <ClInclude Include="src\google\protobuf\stubs\strutil.h" />
+    <ClInclude Include="src\google\protobuf\stubs\time.h" />
+    <ClInclude Include="src\google\protobuf\wire_format_lite.h" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="C:\Users\Adam Heinermann\Downloads\protobuf-cpp-3.6.0\protobuf-3.6.0\install\ZERO_CHECK.vcxproj">
+    <ProjectReference Include="install\ZERO_CHECK.vcxproj">

```

Comment:
can probably get rid of this guy, no idea what it is

Reply:
Yes we should remove that first I guess